src/data/fintype.lean

```diff
@@ -279,6 +279,23 @@ instance {α : Type*} (β : α → Type*)
   fintype.card (sigma β) = univ.sum (λ a, fintype.card (β a)) :=
 card_sigma _ _
 
+instance psigma.fintype {α : Type*} {β : α → Type*} [fintype α] [∀ a, fintype (β a)] : 
```

Comment:
Can these be unified by just using `Sort*` instead of `Type*`?

Reply:
No. `fintype` is only defined for `Type*`, so it would require a case split on which universe it's in, which isn't possible.